### PR TITLE
Try installing the flink CRD

### PR DESCRIPTION
**Pull request by smitelli**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -58,6 +58,10 @@ jobs:
       - name: Install CRDs
         if: steps.list-changed.outputs.changed == 'true'
         run: make crds
+      
+      - name: Check installed crds
+        if: steps.list-changed.outputs.changed == 'true'
+        run: kubectl get crds
```
Comment (diranged) [inline .github/workflows/ci.yaml:64]: ?

Comment (eugenend) [reply .github/workflows/ci.yaml]: this was a test branch to debug the issue, please check out https://github.com/Nextdoor/k8s-charts/pull/140 instead

File: charts/flink-job/templates/flinkjobcluster.yaml [hunk 2]
```diff
@@ -53,7 +53,19 @@ spec:
       {{- toYaml . | nindent 6 }}
       {{- end }}
     {{- end }}
-  # volumeClaimTemplates, initContainers, nodeSelector, tolerations, sideCars not exposed
+    # volumeClaimTemplates, initContainers, sideCars not exposed
+    {{- if .Values.nodeSelector }}
```
Comment (diranged) [inline charts/flink-job/templates/flinkjobcluster.yaml:57]: Make sure to edit `values.yaml` and add the these ettings to the file, along with doc strings...

**Comment by smitelli** [comment]
@eugenend @diranged I'm gonna close this PR as it was just a testing ground.

@eugenend Feel free to integrate anything useful here into the real change at #140.

**Pull request closed without merging**